javascript/sequelize/security/audit/sequelize-raw-query.yaml

```
@@ -0,0 +1,39 @@
+---
+rules:
+  - id: sequelize-raw-query
+    message: >
+       Avoiding SQL string concatenation: untrusted input concatinated with raw
+       SQL query can result in SQL Injection. Data remplacement or data binding
```

```suggestion
       SQL query can result in SQL Injection. Data replacement or data binding
```
great message, just fixing a typo.

---

my bad https://github.com/returntocorp/semgrep-rules/pull/1180